DataCollection/test/Domain/Managers/WaterbirdForagingManagerTests.cs

```diff
@@ -517,5 +517,235 @@ public void DisturbanceCommentsCannotBeLongerThan500Characters()
                 }
             }
         }
+
+        public class CreateFakeSet : Fixture
```

Comment:
@kpunjani example of how to create mock / fake data sets for testing the calls to the interfaces implemented by `DataCollectionContext`.